packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -1869,6 +1883,11 @@ class Scaffold extends StatefulWidget {
   ///    Flutter.
   final String? restorationId;
 
+  /// Builds the [Material] widget used by this widget.
+  ///
```

Comment:
nit: maybe here we should also add that the color of the default `Material` is defined by the provided `backgroundColor`, or if that is null the `scaffoldBackgroundColor` of the nearest `[Theme]`. See [here](https://github.com/flutter/flutter/blob/2524052335ec76bb03e04ede244b071f1b86d190/packages/flutter/lib/src/material/scaffold.dart#L2971)